3.11/data-modeling-collections.md

```diff
@@ -43,19 +43,66 @@ for the entire ArangoDB instance.
 The namespace for collections is shared with [Views](data-modeling-views.html).
 There cannot exist a collection and a View with the same name in the same database.
 
-The collection name needs to be a string that adheres to the following constraints:
+The collection name needs to be a string that adheres to either the **traditional**
+or the **extended** naming constraints. Whether the former or the latter is
+active depends on the `--database.extended-names` startup option.
+The extended naming constraints are used if enabled, allowing many special and
+UTF-8 characters in database, collection, View, and index names. If set to
+`false` (default), the traditional naming constraints are enforced.
 
-- The names must only consist of the letters `A` to `Z` (both in lower 
-  and upper case), the digits `0` to `9`, and underscore (`_`) and dash (`-`)
-  characters. This also means that any non-ASCII names are not allowed.
-
-- Names of user-defined collections must always start with a letter.
-  System collection names must start with an underscore. You should not use
-  system collection names for your own collections.
-
-- The maximum allowed length of a name is 256 bytes.
+{% hint 'info' %}
+The extended naming constraints are an **experimental** feature but they will
+become the norm in a future version. Drivers and client applications
+should be prepared for this feature.
```

Comment:
The sentence "Drivers and client applications should be prepared for this feature" is ambiguous to me.
It can mean both:
1. we expect drivers & apps to be ready for this feature, so it is fine to use it.
2. we expect you to check if drivers & apps are ready, so use it with care.

I guess the intended meaning is that of 2.
So I suggest rephrasing as "Drivers and client applications need to be compatible with this feature. This should be ensured before turning the feature on." or such.